students/hirot_directory/fizz.py

```diff
@@ -0,0 +1,51 @@
+
+'''
+Dev: Hiro Takechi
+Subject: FizzBuzz
+'''
+
+'''
+def print_fizz():
```

Comment:
breaking things down into functions is good in general, but probably not a good idea to do it when they are THIS simple.

Reply:
Though it looks like you didn't end up using them anyway.

Reply:
Hi Chris,

Yes, during Maria's office hours, she taught us to use def by showing the example of grid printer, so I just wanted to try to do the same approach to fizzBuzz but then I couldn't add 'n' in front of "fizz", "buzz" and "fizzBuzz" so I decided to comment them out. 

Thanks,
Hiro